pom.xml

```diff
@@ -90,6 +90,7 @@
         <json.smart.version>2.4.7</json.smart.version>
         <thrift.version>0.13.0</thrift.version>
         <tomcat.version>8.5.65</tomcat.version>
+        <protobuf.version>3.16.1</protobuf.version>
```

Comment:
can we make this 3.19.4 (latest)?
https://nvd.nist.gov/vuln/detail/CVE-2021-22569